test/unit/spec/agent_spec.rb

```diff
@@ -33,8 +33,8 @@
   end
 
   it 'creates lumberjack key and certificate files' do
-    expect(chef_run).to create_file('/etc/lumberjack.key')
-    expect(chef_run).to create_file('/etc/lumberjack.crt')
+    expect(chef_run).to create_file('/opt/logstash/lumberjack.key')
+    expect(chef_run).to create_file('/opt/logstash/lumberjack.crt')
```

Comment:
The agent uses the logstash as the agent by default and not the fowarder.  The fowarder unit test is in https://github.com/rackspace-cookbooks/elkstack/blob/master/test/unit/spec/forwarder_spec.rb#L27-L30